R/utils_data.R

```diff
@@ -62,3 +62,100 @@ column_as_rownames <- function(x, var = "rowname") {
   x[[var]] <- NULL
   x
 }
+
+
+#' Tools for working with column names
+#'
+#' @param x A data frame.
+#' @param row Row to use as column names.
+#' @param na_prefix Prefix to give to the column name if the row has an NA.
+#' Default is 'x', and it will be incremented at each NA (x1, x2, etc.).
```

Comment:
```suggestion
#' @param na_prefix Prefix to give to the column name if the row has an `NA`.
#' Default is 'x', and it will be incremented at each `NA` (`x1`, `x2`, etc.).
```